content/en/real_user_monitoring/guide/monitor-utm-campaigns-in-rum

```diff
@@ -0,0 +1,45 @@
+---
+title: How to Monitor UTM Campaigns in RUM
+kind: guide
+description: Learn how to use RUM & Session Replay to monitor the performance of UTM campaigns.
+---
+
+## Overview
+
+Urchin Tracking Module (UTM) tracking is a parameter that can be added to a URL for tracking the performance of specific campaigns and identifying attribution paths for how visitors arrived on your website. This guide walks you through the types of UTM parameters Datadog RUM collects and how you can use RUM to monitor their use.
+
+## Data collected
+
+UTM campaigns are connected to [View][1] events in RUM. These are collected automatically by the Browser SDK and can be viewed as facets in the RUM explorer. The UTM parameters Datadog collects can be defined as the following:
```

Comment:
@jamiemilstein In the second sentence, can we clarify what "These" refers to? Is it the UTM campaigns or the View events?

Reply:
would "Campaign data" be clear here?

Reply:
Yes, will add this.